client/my-sites/site-settings/redirect-to/README.md

```diff
@@ -0,0 +1,19 @@
+Check and restrict site access rights to the disconnection process.  
+==================================================================
+
+This component redirects to ```/settings/general``` for non-Jetpack
+and Atomic sites. The redirection happens automatically prior to rendering
```

Comment:
I would recommend not adding manual newlines for manual text wrap. Let's leave that to the software that renders the README 😉

Reply:
😂